src/hooks/arbeidsuforhet/useArbeidsuforhetVurderingDocument.ts

```diff
@@ -18,7 +18,7 @@ type ForhandsvarselDocumentValues = {
 
 type AvslagDocumentValues = {
   begrunnelse: string;
-  fom: Date;
+  fom: Date | undefined;
```

Comment:
Må den være sånn fordi den i starten ikke har en dato satt?

Reply:
Det er vel for å kunne forhåndsvise uten å ha fyllt inn dato ja

Reply:
Ja, uten dette feiler forhåndsvisning hvis man ikke har valgt dato. Skjemaet valideres jo, så det hadde ikke vært krise for sending til backend da 🙂